node/config/default.json

```diff
@@ -1,6 +1,8 @@
 {
-    "appSecret": "",
-    "pageAccessToken": "",
-    "validationToken": "",
-    "serverURL": ""
+    "appSecret": "10d0355e8e607a8c5185d713b1686c1f",
```

Comment:
Use this command to add an existing file to your gitignore without deleting it from the remote repo. 

git update-index --assume-unchanged <filename>